gdal/.shed.yml

```diff
@@ -0,0 +1,81 @@
+categories:
+    - Ecology 
+description: Geospatial Data Abstraction Library tools are all dedicated to manipulate raster and vector geospatial data formats.
+long_description: |
+    Gdal is a translator library for raster and vector geospatial data formats that is released under an X/MIT style Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single raster abstract data model and single vector abstract data model to the calling application for all supported formats. It also comes with a variety of useful command line utilities for data translation and processing.
+
+homepage_url: https://www.gdal.org/
+name: gdal
+owner: mnhn65mo
+remote_repository_url: https://github.com/Alanamosse/Galaxy-E/tree/master/tools/gdal
+type: unrestricted
+
+repositories:
+  gdaladdo:
```

Comment:
can we keep all thos following the same naming schema, e.g. `gdal_xxx`

Reply:
The general problem with this very explicit approach is that I foresee that we forget to add files to a repo. But the tests fill pass the TS package can be broken.

Reply:
Ok with your first comment.

For the 2nd, what it your recommendation?  
Use `auto_tool_repositories` ?